versions.sh

```diff
@@ -26,5 +26,6 @@ meteor_versions=( \
 	'2.3.5' \
 	'2.3.6' \
 	'2.4' \
-	'2.5'
+	'2.5' \
+	'2.5.6'
```

Comment:
Please comments for the skipped versions (see 1.10 above).

Reply:
Done. https://github.com/disney/meteor-base/pull/100/commits/8c0306f2ed5152a448609e854ee5482481b2bd88